backend/inspirehep/records/config.py

```diff
@@ -313,7 +313,40 @@
         "search_as_you_type": {
             "experiment": {
                 "_source": ["legacy_name", "control_number", "self"],
-                "field": "experiment_search_as_you_type",
+                "fields": [
+                    {
+                        "field": "accelerator_search_as_you_type",
+                        "boost": 1,
+                    },
+                    {
+                        "field": "collaboration_search_as_you_type",
+                        "boost": 1,
+                    },
+                    {
+                        "field": "experiment_short_name_search_as_you_type",
+                        "boost": 1,
```

Comment:
maybe it would make sense to boost this too? how have you arrived at the current weights?

Reply:
I took them from the `completion` weights https://github.com/inspirehep/inspirehep/commit/bd2d14ac715df1ae6ee241a76659fff63983636d#diff-4bd65a4802a80112f9f39963c9487595be8604806969c45388db10686daed263R32-R43. I don't really know which ones to boost since I don't know how it's been used from the curators, any suggestions are welcome

Reply:
ok, let's start with this and tune it based on feedback.